ui/src/pages/HomePage.tsx

```diff
@@ -4,16 +4,83 @@ import React from "react";
 export const HomePage: React.FC = () => {
 
   return <>
-    <Card title="Example project for visualisation with influxDB backend">
-      This project is example how to use influxdata as source for different graph/visualisation libraries. 
-      <div />
-      Function "useInfluxSource" simulates influxdb influx source through csv influx together with user inputed select of which columns is selected to group values. UI has also buildin generator for random csv lines.
-      <div />
-      Only not-pivoted data (values stored in "_value" column) are supported for now.
-      <div />
-      <strong>normalizedDataFromTable</strong> show proposed function for easier more "JavaScript-ish" working with incoming influx data. There should be only simple transformation needed after data normalization function.
-      <div />
-      Simplification algorithm presented on Best practice tab is <strong>not</strong> used for libraries (original data is used)
+    <Card title="Visualize InfluxDB query results by using the 3rd party React components">
+
+      <div>
+
+        <p>This project is a <strong>playground for React front
+        developers</strong> that look for ways to visualize
+        <strong>InfluxDB</strong> query results by using the 3rd party
```

Comment:
Some strong text has no space around ("visualizeInfluxDB" instead of "visualize InfluxDB")